tfhe/src/high_level_api/integers/keys.rs

```diff
@@ -119,6 +119,11 @@ impl IntegerServerKey {
         let Some(cks) = &client_key.key else {
             return Self::default();
         };
+        assert_eq!(
+            cks.parameters().message_modulus().0,
+            1 << 2,
+            "This API only supports integers with 2 bits per block (MessageModulus(16))",
```

Comment:
MessageModulus(4) ? in that case you could use the debug format to avoid having to write it yourself :)